python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -221,7 +231,7 @@ def _task_done_callback(actor_to_return):
             self._submit_data_task(
                 gen,
                 bundle,
-                lambda: _task_done_callback(actor_to_return),
+                lambda: _task_done_callback(actor_to_return),  # noqa: B023
```

Comment:
this change seems unrelated?

Reply:
Yes, was suppressing a warning with ./scripts/format.sh